third_party/libgcc/arm/lib1funcs.S

```diff
@@ -1465,7 +1465,8 @@ LSYM(Lover12):
         asr     ah, r2
         sub     r2, #32
         @ If r2 is negative at this point the following step would OR
-        @ the sign bit into all of AL.  That's not what we want...
+        @ <bruening: removing quote on next line since one toolchain trips over it>
+        @ the sign bit into all of AL.  Thats not what we want...
```

Comment:
s/Thats/That is/
or
s/Thats not what we want/We do not want that/

Is the comment in the above line intended to be kept?

Reply:
> Is the comment in the above line intended to be kept?

Yes, so future devs aren't confused by the diff vs upstream lib1funcs.S.  Looks like I should mention this or include the diff in ../README.dynamorio too: will do that.

Reply:
Maybe just the mention in the README is enough and we don't need the embedded comment.